### ISSUE TITLE
matchMS fails if msp file contains tabs.

### ISSUE THREAD

**Issue by hechth** [RECETOX/galaxytools#99]
Matching spectral libraries with matchMS fails if the `.msp` file contains tabstop separated values.

See error message below:
```
Traceback (most recent call last):
  File "/mnt/volume/shared/galaxy/var/shed_tools/testtoolshed.g2.bx.psu.edu/repos/recetox/matchms/6a736abe431f/matchms/matchms_wrapper.py", line 60, in <module>
    main(argv=sys.argv[1:])
  File "/mnt/volume/shared/galaxy/var/shed_tools/testtoolshed.g2.bx.psu.edu/repos/recetox/matchms/6a736abe431f/matchms/matchms_wrapper.py", line 41, in main
    reference_spectra = [
  File "/mnt/volume/shared/galaxy/var/shed_tools/testtoolshed.g2.bx.psu.edu/repos/recetox/matchms/6a736abe431f/matchms/matchms_wrapper.py", line 41, in <listcomp>
    reference_spectra = [
  File "/mnt/volume/shared/galaxy/var/dependencies/_conda/envs/mulled-v1-9acac30a99c41c49633116b3db32cd657baeb355a349a258bc3d42833c66c0fe/lib/python3.8/site-packages/matchms/importing/load_from_msp.py", line 87, in load_from_msp
    for spectrum in parse_msp_file(filename):
  File "/mnt/volume/shared/galaxy/var/dependencies/_conda/envs/mulled-v1-9acac30a99c41c49633116b3db32cd657baeb355a349a258bc3d42833c66c0fe/lib/python3.8/site-packages/matchms/importing/load_from_msp.py", line 43, in parse_msp_file
    masses.append(float(splitted_line[0]))
ValueError: could not convert string to float: '116.05576\t29277'
```

**Comment by hechth** [comment]
Since this appears in various version I've opened an issue in the original repository [here](https://github.com/matchms/matchms/issues/192)

**Comment by hechth** [comment]
Will be fixed once we upgrade to 0.8.2, but the package isn't available in conda yet.
